plugins/content/caption/base/lib/api/KalturaCaptionAsset.php

```diff
@@ -61,6 +61,24 @@ class KalturaCaptionAsset extends KalturaAsset
 	 */
 	public $status;
 	
+	/**
+	 * The parent id of the asset
+	 *
+	 *@var string
+	 *@readonly
+	 *
+	 */
+	public $parentId;
+	
+	/**
+	 * Is multi-language caption asset
+	 *
+	 *@var bool
+	 *@readonly
+	 *
+	 */
+	public $isMulti;	
```

Comment:
@tan-tan-kanarek regarding you last comment for updating the parent id field.
Would we like to enable suers to update parent id via the API? If not maybe we should add requires permission on this field and add the permission only to the batch partner